modules/video/consts/Video.ts

```diff
@@ -0,0 +1,13 @@
+type Video = {
```

Comment:
Can you add the OpenAPI definition to this type that way it can be referenced in the route as a schema.